pkg/ruler/ruler.go

```diff
@@ -149,59 +139,64 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.DurationVar(&cfg.ResendDelay, "ruler.resend-delay", time.Minute, `Minimum amount of time to wait before resending an alert to Alertmanager.`)
 }
 
+// MultiTenantManager is the interface of interaction with a Manager that is tenant aware.
```

Comment:
```suggestion
// MultiTenantManager interface describes interaction between Ruler, that periodically 
// loads rules from the store, and object that keeps rules in memory for each tenant.
```

It sounds to me like "multi-tenant aware rules manager" :)

Reply:
> and object that keeps rules in memory for each tenant

The `DefaultMultiTenantManager` uses a mapper to write rules to a file and passes file path to the Prometheus Manager. Maybe we can modify this to

```
// MultiTenantManager interface describes interaction between Ruler, that periodically 
// loads rules from the store, and object that syncs rules to the Prometheus manager
// for every tenant.
```